DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Dashboard/_DashboardTopCardGroup.cshtml

```diff
@@ -1,5 +1,6 @@
-﻿@inject IConfiguration Configuration
+﻿@inject IConfiguration configuration

```

Comment:
Don't think we are using the configuration in this file anymore, so we can get rid of it.